src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -306,7 +306,7 @@ export default class PhotoField extends React.Component {
     const fileTypes = this.props.uiSchema['ui:options'].fileTypes;
     if (!isValidFileType(file.name, fileTypes)) {
       this.props.onChange({
-        errorMessage: 'We weren’t able to upload your file. Please make sure the file you’re uploading is a jpeg, .png, .tiff,  or .bmp file and try again.'
+        errorMessage: 'We weren’t able to upload your file. Please make sure the file you’re uploading is a gif, jpeg, .png, or .tiff file and try again.'
```

Comment:
Are we missing a . in front of gif?

Reply:
I think the lack of one in front of jpeg is because saying "jpeg" implies both extensions. I'm going to use that for tiff, too, and left the `.` in front of the others.